translators/apis.py

```diff
@@ -160,6 +160,7 @@ def __init__(self):
         self.get_ip_url = 'https://httpbin.org/ip'
         self.ip_api_addr_url = 'http://ip-api.com/json'  # must http.
         self.ip_tb_add_url = 'https://ip.taobao.com/outGetIpInfo'
+        self.default_country = os.environ.get('TRANSLATORS_DEFAULT_COUNTRY', None)
```

Comment:
Good idea. I didn't see your merge beforehand, please resubmit it on the new version.